documentation/source/user_section/tutorials/customizing_registration.rst

```diff
@@ -0,0 +1,67 @@
+
+While the default usage of ``sct_register_to_template`` is simple enough, the underlying command provides many options to adapt the registration process to your specific data and pipeline. The subsections below provide an overview of common tweaks to the command.
+
+Because choosing the right configuration for your data can be overwhelming, SCT provides a forum where you can ask for clarification and guidance.
+
+The ``-param`` argument
+***********************
+
+The flag -param lets you select registration parameters at each step. Below is a sample input for ``-param``:
+
+.. code-block::
+
+   # Note: Command has been split up for readability. Normally, you would input this with no line breaks.
+   -param step=0,type=label,dof=Tx_Ty_Tz_Sz:
+          step=1,type=imseg,algo=centermassrot,metric=MeanSquares,iter=10,smooth=0,gradStep=0.5,slicewise=0,smoothWarpXY=2,pca_eigenratio_th=1.6:
+          step=2,type=seg,algo=bsplinesyn,metric=MeanSquares,iter=3,smooth=1,gradStep=0.5,slicewise=0,smoothWarpXY=2,pca_eigenratio_th=1.6
+
+This long string of values defines a 3-step transformation. Each step is separated by a ``:`` character, and begins with ``step=#``, where ``#`` can be ``0, 1, 2, etc``. At each step, a distinct transformation is computed:
+
+* **Step 0:** Straighten the spinal cord, then match the subject labels to the template labels.
+* **Step 1:** Nonrigid deformation, first pass. Deals with large deformations in the spinal cord.
+* **Step 2:** Nonrigid deformation, second pass. Applies fine cord shape adjustments.
+
+Typically, step 0 is not altered. However, Steps 1 and 2 can be tweaked, and additional steps (e.g. 3, 4) can be added. Some of the common parameters to tweak include:
+
+   .. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/registration_to_template/sct_register_to_template-param-algo.png
+      :align: right
+      :figwidth: 40%
+
+      Visualization of algorithms to choose from for the ``algo`` parameter of ``-param``.
+
+* ``algo`` This is the algorithm used to compute the nonrigid deformation. Choice of algorithm depends on how coarse/fine you want your transformation to be. This depends on which step you are modifying (Step 1, step 2, step 3, etc.) as well as the nature of the spinal cord you are working with.
+
+   - **translation**: axial translation (x-y)
+   - **rigid**: transaltion + rotation about z axis
+   - **affine**:
+   - **b-splinesyn**: based on ants binaries
+   - **syn**: (not bspline regularized)
+   - **slicereg**: ANTs+Us, regualarized translation across slices that is regularized across S-I (used with segmentation for a pre-alignment cord centerline + template cord centerline -- basically alignment of the core centerline)
+   - **centermassrot**: similar to slicereg, center of mass of segmentation on each slice and then align with center of mass in the template space, also does a rotation in case your subject turned neck or sometimes compression
```

Comment:
```suggestion
   - **centermassrot**: Compute the center of mass of the source and destination segmentation and align them slice-by-slice. This algorithm also estimates the orientation of the cord in the axial plane and matches that of the destination cord. This feature could be useful in case the subject rotated their neck, or if the cord is rotated due to a disc compression. Note that this algorithm only works with segmentations (contrary to slicereg). 
```